<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>571</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <property name="accessibleName">
      <string/>
     </property>
     <attribute name="title">
      <string>Add Inventory</string>
     </attribute>
     <widget class="QLineEdit" name="addproductName">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>30</y>
        <width>251</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Product Name</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Product Category</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="addproductCount">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>110</y>
        <width>251</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>116</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Product Count</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>150</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Storage Facility</string>
      </property>
     </widget>
     <widget class="QComboBox" name="addproductStorage">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>150</y>
        <width>251</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="addproductCategory">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>70</y>
        <width>251</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="addtoinventoryButton">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>410</y>
        <width>121</width>
        <height>33</height>
       </rect>
      </property>
      <property name="text">
       <string>Add to Inventory</string>
      </property>
     </widget>
     <widget class="QLabel" name="addproductAlert">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>480</y>
        <width>441</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QCheckBox" name="addproductPaymentis">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>360</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Payment Sent</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="addproductcustName">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>230</y>
        <width>251</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_19">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>310</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Total Cost</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="addproductcustAdd">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>270</y>
        <width>251</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_20">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>270</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Buyer Address</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_22">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>190</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Buyer Information :</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="addproductCost">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>310</y>
        <width>251</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_23">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>230</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Buyer Name</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Sell Inventory</string>
     </attribute>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Product Name</string>
      </property>
     </widget>
     <widget class="QComboBox" name="sellproductName">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>30</y>
        <width>251</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>111</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Product Category</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="sellproductCategory">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>70</y>
        <width>251</width>
        <height>31</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QSpinBox" name="sellproductCount">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>110</y>
        <width>251</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Item Count</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>170</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Customer Information :</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="sellproductCustName">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>210</y>
        <width>251</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>210</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Customer Name</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>260</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Customer Address</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="sellproductCustAddress">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>250</y>
        <width>251</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>300</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Total Cost</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="sellproductCost">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>290</y>
        <width>251</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="sellproductButton">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>380</y>
        <width>131</width>
        <height>33</height>
       </rect>
      </property>
      <property name="text">
       <string>Sell Inventory</string>
      </property>
     </widget>
     <widget class="QLabel" name="sellproductAlert">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>410</y>
        <width>441</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QCheckBox" name="sellpaymentCheck">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>340</y>
        <width>251</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Payment Received</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_5">
     <attribute name="title">
      <string>Payments</string>
     </attribute>
     <widget class="QLabel" name="label_18">
      <property name="geometry">
       <rect>
        <x>18</x>
        <y>29</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Invoice Number</string>
      </property>
     </widget>
     <widget class="QPushButton" name="paymentsAdd">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>80</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Add Payment</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="paymentinvoiceNumber">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>30</y>
        <width>251</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="paymentAlert">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>150</y>
        <width>281</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Reports</string>
     </attribute>
     <widget class="QLineEdit" name="reportsearchText">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>30</y>
        <width>251</width>
        <height>31</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Enter Name or Category</string>
      </property>
     </widget>
     <widget class="QPushButton" name="reportsearchButton">
      <property name="geometry">
       <rect>
        <x>283</x>
        <y>80</y>
        <width>121</width>
        <height>33</height>
       </rect>
      </property>
      <property name="text">
       <string>Show Inventory</string>
      </property>
     </widget>
     <widget class="QPushButton" name="reportsshowpendingPayments">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>140</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Show Pending Payments</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Settings</string>
     </attribute>
     <widget class="QLineEdit" name="settingsCompanyName">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>30</y>
        <width>251</width>
        <height>31</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>17</x>
        <y>29</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Company Name</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="settingsCompanyAddress">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>70</y>
        <width>251</width>
        <height>31</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>17</x>
        <y>69</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Company Address</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="settingsCompanyPhone">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>110</y>
        <width>251</width>
        <height>31</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>110</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Company Phone</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>150</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Tax Percent</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="settingsPercent">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>150</y>
        <width>251</width>
        <height>31</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="settingsSave">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>250</y>
        <width>111</width>
        <height>33</height>
       </rect>
      </property>
      <property name="text">
       <string>Save Settings</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="settingsProductGroup">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>190</y>
        <width>251</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>190</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Add Product Group</string>
      </property>
     </widget>
     <widget class="QLabel" name="settingsAlert">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>310</y>
        <width>211</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
